### PR TITLE
Lists added using different endpoints. Trailer feature fixed.

### PR DESCRIPTION
**Pull request by rafapirotto**
image

### CODE REVIEW

**Comment by rafapirotto** [comment]
image

File: src/app/Components/Movie/index.jsx [hunk 1]
```diff
@@ -4,14 +4,51 @@ import PropTypes from 'prop-types';
...
+  const left = 100 - progress;
+
+  return (
+    <div className="flex mt-1">
+      <div
+        style={{
+          background: 'linear-gradient(360deg, #0578FF 0%, #0578FF 100%)',
+          borderRadius: '100px 0px 0px 100px',
+          height: '7px',
+          width: `${progress}%`,
```
Comment (ceciliagu) [inline src/app/Components/Movie/index.jsx]: Why not extracting this to a stylesheet?

File: src/app/Components/Movie/index.jsx [hunk 2]
```diff
@@ -4,14 +4,51 @@ import PropTypes from 'prop-types';
...
 
-const Movie = ({ url, title }) => (
-  <div className="mr-1 mb-4">
-    <img src={url} alt={title} />
-  </div>
+const defaultProps = {
+  watchingProgress: null,
+};
+
+const randomIntegerBetween = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
```
Comment (ceciliagu) [inline src/app/Components/Movie/index.jsx]: Maybe it's better to import this function from `utils`

File: src/app/Containers/MyListContainer/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { MovieList } from '../../Components';
+import { getMyList } from './duck/operations';
+
+const MyListContainer = () => {
+  const dispatch = useDispatch();
+  const selector = useSelector((state) => state.myListMovies);
```
Comment (ceciliagu) [inline src/app/Containers/MyListContainer/index.js:9]: You can destructure selector right here

**Review by ceciliagu — COMMENTED** [review]
Remember the WIP commits are only meant to be temporary

**Review by ceciliagu — APPROVED** [review]

**Pull request merged**